wdl/ExpansionHunter.wdl

```diff
@@ -142,3 +165,145 @@ task RunExpansionHunter {
         maxRetries: runtime_attr.max_retries
     }
 }
+
+task SplitVariantCatalog {
+    input {
+        File variant_catalog
+        Int batch_size
+        String? split_variant_catalog_docker
+        String? output_prefix
+        RuntimeAttr? runtime_attr_override
+    }
+
+    output {
+        Array[File] catalogs_json = glob("${output_prefix_}*.json")
+    }
+
+    String output_prefix_ =
+        if defined(output_prefix) then
+            select_first([output_prefix])
+        else
+            "output_"
+
+    String split_variant_catalog_docker_ =
+        if defined(split_variant_catalog_docker) then
+            select_first([split_variant_catalog_docker])
+        else
+            "python:slim-buster"
```

Comment:
See comment about `variant_catalog_batch_size_` for a shorter way to write these.

Reply:
Thanks!